packages/adminrouter/extra/src/includes/metrics-http-level.conf

```diff
@@ -1,15 +1,18 @@
 # Output fine-grained http directive level metrics using the VTS module.
-# 
+#
 # Caution:
 # A request which matches a location that has a VTS module filter defined will be
 # counted on the Nginx server and location directive level filter as well.
 #
 # This only exists in order for the Nginx VTS module to calculate the average latency
 # per second over all requests for us. This could potentially be removed once we
 # calculate the average latency ourselves in Grafana.
-# 
+#
 # By supplying no value to http:= and all:= the keys will only be used by our
 # custom relabelling mechanism the metric but will result in empty labels http
 # and all. Empty labels are dropped by Prometheus and thus they will not show
 # up in Grafana.
 vhost_traffic_status_filter_by_set_key http:= all:=;
+
+# Set the buckets for measuring request times.
+vhost_traffic_status_histogram_buckets 0.008 0.04 0.2 1 5 25;
```

Comment:
Any particular reason you put this under the first `filter_by_set_key`?
(I know there is no difference after all.)

Reply:
No, just appending it to the file.